Assets/_Application/Projects/PrefabLibrary.cs

```diff
@@ -77,5 +81,20 @@ private LayerGameObject FindPrefabInGroup(string id, PrefabGroup group)
         {
             return group.prefabs.FirstOrDefault(prefab => prefab.PrefabIdentifier == id);
         }
+
+        public LayerGameObject FindPrefabOfType<T>() where T : LayerGameObject
```

Comment:
Dit is niet veilig - er zijn meerdere prefabs met dezelfde LayerGameObject. Het type layergameobject is geen identificatie. 

Ik herinner mij niet waar ik dit gebruikt zie worden in de code - maar geef dan een List terug of noem de functie FindFirstPrefabOfType zodat in de aanroepende code duidelijk is dat dit geen uniek criteria is